### ISSUE TITLE
Can't set backgroundColor of header

### ISSUE THREAD

**Issue by darkwood-studios** [TheJuki/KFormMaster#65]
```
header {
    title = "test"
    backgroundColor = Color.WHITE
}
```

"Unresolved Reference: backgroundColor"

**Comment by TheJuki** [comment]
I just released [5.0.7](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if backgroundColor is working now.

**Comment by TheJuki** [comment]
I am closing this issue. If you have any issues feel free to reopen.
